hr_holidays_validator_list/models/hr_employee.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import fields, models
+
+
+class HrEmployeeBase(models.AbstractModel):
+    _inherit = "hr.employee.base"
+
+    leave_manager_ids = fields.Many2many(
+        "res.users",
+        string="Time Off Managers",
+        store=True,
+        readonly=False,
+        help="""Select the users responsible for approving 'Time Off' of this employee.\n
```

Comment:
This double the end line as `"""` will preserve the return line and the `\n` is added. Is it intended?